docs/iidm/model/twoWindingsTransformer.md

```diff
@@ -5,28 +5,64 @@ layout: default
 
 The `com.powsybl.iidm.network.TwoWindingsTransformer` interface is used to model a two windings power transformer.
 A two windings power transformer is connected to two voltage levels (side 1 and side 2) that belong to a same substation.
-A [ratio tap changer]() or a [phase tap changer]() can be associated to a two windings power transformer.
+A [ratio tap changer](./ratioTapChanger.md) and/or a [phase tap changer](./phaseTapChanger.md) can be associated to a two windings power transformer.
 
 # Characteristics
 
-| Attribute | Type | Required | Default value | Description |
-| --------- | ---- | -------- | ------------- | ----------- |
-| R | double | yes | - | The nominal series resistance |
-| X | double | yes | - | The nominal series reactance |
-| G | double | yes | - | The nominal magnetizing conductance |
-| B | double | yes | - | The nominal magnetizing susceptance |
-| RatedU1 | double | yes | - | The primary winding rated voltage |
-| RatedU2 | double | yes | - | The secondary winding rated voltage |
+| Attribute | Type | Unit | Required | Default value | Description |
+| --------- | ---- | ---- | -------- | ------------- | ----------- |
+| id | string | - | yes | - | Unique identifier of the transformer |
+| name | string | - | yes | - | Humanly readable name of the transformer |
+| $$r_{nom}$$ | double | $$\Omega$$  | yes | - | The nominal series resistance at the secondary side of the transformer |
+| $$x_{nom}$$ | double | $$\Omega$$ | yes | - | The nominal series reactance at the secondary side of the transformer |
+| $$g_{nom}$$ | double | S | yes | - | The nominal magnetizing conductance at the secondary side of the transformer |
+| $$b_{nom}$$ | double | S | yes | - | The nominal magnetizing susceptance at the secondary side of the transformer |
+| $$V_{1\ nom}$$ | double | kV | yes | - | The primary winding rated voltage |
+| $$V_{2\ nom}$$ | double | kV | yes | - | The secondary winding rated voltage |
 
-B, G, R and X shall be specified at the secondary voltage side.
+# Model
+Two windings transformer are modelled with the following equivalent $$\pi$$ model:
+
+![Power line model](./images/two-windings-transformer-model.svg){: width="50%" .center-image}
+
+With series impedance $$z$$ and the shunt admittance $$y$$, at secondary side of the transformer, and $$\rho$$ the voltage ratio and $$\alpha$$ the angle difference:
+
+$$
+r=r_{nom}.\left(1+\frac{r_{r, tap} + r_{\phi, tap}}{100}\right)\\
```

Comment:
explain here what is r_r and r_phi. These values are not used before this equation (same comments for all following equations)